18F/guides
# Add introduction page for v2 De-risking Guide

## Changes proposed in this pull request:

Add the structure and a first page for the v2 De-risking Government Tech guide to the `derisking-government-tech` feature branch.

- Create file structure and navigation for new v2 De-risking Guide
- Add introduction page content and styling
- Stub out additional pages for 1st and 2nd level navigation

### Notes on implementation

- I've duplicated the `chapter_image.html` include from the v1 De-risking Guide, since the plan is to remove the original guide once v2 has been up for several months.
- The 2x2 "What's in this guide" section is implemented with the [USWDS card component](https://designsystem.digital.gov/components/card/). I added SCSS styles for layout settings that I couldn't solve with USWDS utility classes.

## testing

Beyond the automated validations on the PR, manually test the following on a PR build:

- Visit the `/derisking-government-tech/` route on the PR build URL and confirm it loads the Introduction page: https://federalist-8bc0b42c-5fd0-4ae8-9366-cd9c265a4446.sites.pages.cloud.gov/preview/18f/guides/drg-v2-add-introduction-page/derisking-government-tech/
- Confirm that the content, layout, and images on the intro page look correct
- Shake out the page's responsiveness when the browser window is resized
- Check mobile device rendering using browser dev tools
- Confirm meta title, meta description, and social meta tags are present in the Introduction page source
- Shake out the top and left nav (additional pages should exist but contain only titles and "TK")

## security considerations

No security considerations: These updates use the existing framework and patterns of the 18F Guides site.
